src/KernelIntegrator/SPHIntegrationKernel.hh

```diff
@@ -22,7 +22,9 @@ public:
   
   SPHIntegrationKernel(const TableKernel<Dimension>& kernel);
 
-  virtual double extent(const Scalar Hmult) const { return mKernel.kernelExtent() / Hmult; }
+  virtual double extent(const Scalar Hmult) const override {
```

Comment:
Same comment as my prior one: adding override is great, but I don't want to mix up styles by breaking up this line.

Reply:
Fixed.